mkosi/__init__.py

```diff
@@ -608,21 +612,22 @@ def run_build_scripts(context: Context) -> None:
                     stdin=sys.stdin,
                     sandbox=context.sandbox(
                         network=context.config.with_network,
-                        options=sources + [
-                            "--ro-bind", script, "/work/build-script",
-                            "--ro-bind", json, "/work/config.json",
-                            "--ro-bind", cd, "/work/scripts",
-                            "--bind", context.root, context.root,
-                            "--bind", context.install_dir, "/work/dest",
-                            "--bind", context.staging, "/work/out",
+                        mounts=[
+                            *sources,
+                            Mount(script, "/work/build-script", ro=True),
+                            Mount(json, "/work/config.json", ro=True),
+                            Mount(cd, "/work/scripts", ro=True),
+                            Mount(context.root, context.root),
+                            Mount(context.install_dir, "/work/dest"),
+                            Mount(context.staging, "/work/out"),
                             *(
-                                ["--bind", os.fspath(context.config.build_dir), "/work/build"]
+                                [Mount(os.fspath(context.config.build_dir), "/work/build")]
```

Comment:
That's no longer needed
```suggestion
                                [Mount(context.config.build_dir, "/work/build")]
```